app/templates/grunt/_package.json

```diff
@@ -39,6 +42,7 @@
     "grunt-string-replace": "1.2.0",
     "grunt-topdoc": "0.3.0",
     "jit-grunt": "0.9.1",
+    "parallelshell": "2.0.0",
```

Comment:
Is `parallelshell` the kind of thing we should encourage people to install globally?